cron/hourly/git_maintenance_tasks_lock

```diff
@@ -4,4 +4,5 @@ set -e
 # Run git_maintenance_tasks only if not running already
 DIR="$(readlink -e "$(dirname $0)/..")"
 mkdir -p ~/lmms-repo
-flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock "$DIR/git_maintenance_tasks"
+flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock \
+	"$DIR/git_maintenance_tasks_retry"
```

Comment:
>the new instance would mess with the running one

The purpose of `flock` is to avoid starting the new job if another one is running. If `flock` fails because of a running instance, `rm` will wipe out the working repository.

>What's the difference exactly?

`git_maintenance_tasks` is the regular process. `git_maintenance_tasks_retry` is the retry strategy. Using separate scripts helps with debugging and improvements. One script should do one job and do it well. Technically, you can merge the scripts, but with a maintenance penalty; do you prefer that?